src/components/AuthWrapper/AuthWrapper.tsx

```diff
@@ -39,6 +40,8 @@ export const AuthWrapper: FC<Props> = props => {
                     <ConfirmSignUp authProps={props} path="/confirm" />
                 )}
                 <ConfirmSignUp authProps={props} path="/confirm/:code" />
+
+                <ApplicationPage path="/application/:id" />
```

Comment:
This doesn't belong here. This component is for wrapping all the authentication stuff.